experimental/examples/bubblebench/editable-shared-tree/README.md

```diff
@@ -0,0 +1,35 @@
+# @fluid-example/bubblebench-sharedtree
+
+## Getting Started
```

Comment:
This readme is where you should document what this package does. Specifically why it exists. By comparing the descriptions in this readme with the corresponding one that should be in the other bubble bench versions I should be able to figure out that this uses the editableTree API from `tree` and exists to measure the over head of that compared to other approaches.